src/manual/coupon-details.md

```diff
@@ -9,58 +9,71 @@ Label (required)
+: This field contains the case-sensitive coupon provider class name, e.g. "FixedRebate". A list of available providers is shown when you click into the input field. Selecting a provider adds it to the input field. Furthermore, each provider can be enhanced by one or more decorators, which can be choosen from the drop-down list that appears when the "+" symbol next to the input field is clicked. Selecting a decorator adds it sequentially to the provider or any other previously added decorator(s).  
+For a detailed description of all the available coupon providers and decorators provided by the Aimeos core, have a look at [Available coupons](coupons.md) and [Adding features](coupon-decorators.md).
 
 Start date (optional)
-: The point in time when the coupon will be available for customers to enter their codes. The date/time has to be in ISO format (YYYY-MM-DD HH:mm:ss) and the hours must be in the range of 0-23.
+: The point in time when the coupon will be available for customers to enter their codes. The date/time value has to be in ISO format (YYYY-MM-DD HH:mm:ss) and the hours must be in the range of 0-23.
 
 End date (optional)
-: The point in time when the coupon won't be available any more. For this field the same format applies as for "Start date".
+: The point in time when the coupon won't be available any more. The same format rules apply as for "Start date".
 
+!!! note
+     Keep in mind that each decorator can bring along its own configuration setting(s). Therefore, depending on which decorator you activate and how many decorators you implement, the amount of configuration fields on the right site / in the lower section of the detail page will change (and therefore also differ from the screenshot shown above).
+
+!!! hint
+    Up to and including *Aimeos* 20.10.2, uninstalling a decorator has to be done manually by drag-selecting the decorator name including its preceding comma, hitting the delete key or cutting out the selected text and saving afterwards.  
+    Starting with *Aimeos* 20.10.3, this field is read only. The only way to delete a decorator now is by replacing the current configuration completely via adding the provider anew and re-assigning decorators again.
+    
+    A decorator's configuration field(s) will not be deleted right away, which might be helpful for testing purposes. If you don't know, which configuration fields do not belong to the current setup anymore, simply hit *Save* in the top right corner of the page: the page will refuse to save and instead warn about all superfluous fields.
```

Comment:
This is not true. You can save without removing the superfluous fields. This is the reason why it's helpful for testing